### PR TITLE
fix: remove fake window which the trick way to open setting

### PR DESCRIPTION
**Pull request by AkaShark**
closed https://github.com/tisfeng/Easydict/issues/528#issuecomment-2089805673

### CODE REVIEW

File: Easydict/Swift/View/MenuItemView.swift [hunk 1]
```diff
@@ -67,10 +67,19 @@ struct MenuItemView: View {
                 latestVersion = version
             }
         }
+        .openSettingsAccess()
```
Comment (phlpsong) [inline Easydict/Swift/View/MenuItemView.swift:70]: According to the doc of `SettingsAcess`, this should not used in menu content.

> If using a menu-based MenuBarExtra, do not apply openSettingsAccess() to the menu content. openSettings() cannot be used there due to limitations of SwiftUI.

https://github.com/orchetect/SettingsAccess?tab=readme-ov-file#2-use-in-a-menubarextra-menu

**Comment by phlpsong** [comment]
Screenshot 2024-05-09 at 10 52 16

The menu UI looks wired after this change.

**Comment by AkaShark** [comment]
> Screenshot 2024-05-09 at 10 52 16
> The menu UI looks wired after this change.

You are right, let me change the implementation

**Comment by phlpsong** [comment]
This works well for me.👍

**Review by phlpsong — DISMISSED** [review]

**Review by tisfeng — APPROVED** [review]
Looks good.

**Review by phlpsong — APPROVED** [review]

**Review by Jerry23011 — APPROVED** [review]

**Pull request merged**